tap_hubspot_beta/streams.py

```diff
@@ -1689,4 +1689,78 @@ class AssociationTasksDealsStream(AssociationTasksStream):
     """Association Tasks -> Deals Stream"""
 
     name = "associations_tasks_deals"
-    path = "crm/v4/associations/tasks/deals/batch/read"
\ No newline at end of file
+    path = "crm/v4/associations/tasks/deals/batch/read"
+
+
+class DiscoverCustomObjectsStream(hubspotV3Stream):
+    name = "discover_stream"
+    path = "crm-object-schemas/v3/schemas"
+    primary_keys = ["id"]
+    replication_key = "updatedAt"
+
+    schema = th.PropertiesList(
+        th.Property("labels", th.ObjectType(
+            th.Property("singular", th.StringType),
+            th.Property("plural", th.StringType)
+        )),
+        th.Property("requiredProperties", th.ArrayType(th.StringType)),
+        th.Property("searchableProperties", th.ArrayType(th.StringType)),
+        th.Property("primaryDisplayProperty", th.StringType),
+        th.Property("secondaryDisplayProperties", th.ArrayType(th.StringType)),
+        th.Property("description", th.StringType),
+        th.Property("archived", th.BooleanType),
+        th.Property("restorable", th.BooleanType),
+        th.Property("metaType", th.StringType),
+        th.Property("id", th.StringType),
+        th.Property("fullyQualifiedName", th.StringType),
+        th.Property("createdAt", th.DateTimeType),
+        th.Property("updatedAt", th.DateTimeType),
+        th.Property("createdByUserId", th.IntegerType),
+        th.Property("updatedByUserId", th.IntegerType),
+        th.Property("objectTypeId", th.StringType),
+        th.Property("properties", th.ArrayType(th.ObjectType(
```

Comment:
`showCurrencySymbol` (Boolean) is missing, `hubspotDefined` (Boolean) is missing, `createdAt` (DateTimeType) is missing,  `updatedAt` (DateTimeType) is missing, `archivedAt` (DateTimeType) is missing, `referencedObjectType` (String) is missing, `calculationFormula` (String) is missing
*Check the response here*: https://api.hubapi.com/crm-object-schemas/v3/schemas